.github/workflows/ci.yml

```diff
@@ -39,10 +39,12 @@ jobs:
           key: ${{ runner.os }}-pip-${{ env.PYTHON_VERSION }}-${{ hashFiles('**/requirements.txt') }}
           restore-keys: ${{ runner.os }}-pip-${{ env.PYTHON_VERSION }}-${{ hashFiles('**/requirements.txt') }}
 
+      # This is needed otherwise pip warns:
+      # Using legacy 'setup.py install' for DEP, since package 'wheel' is not installed.
       - run: pip install --upgrade wheel
 
       - name: Install Python dependencies
-        run: python3 -m pip install -r requirements.txt
+        run: pip install -r requirements.txt
```

Comment:
Not sure why this was explicitly added in the past, but since we already set the Python version to 3.7, python3 should be already in use.